shiny-ci-cd/actions/ci-cd-renv.yml

```diff
@@ -8,17 +8,32 @@ name: CI-CD
 
 jobs:
   CI-CD:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.config.os }}
```

Comment:
Can you add a comment here explaining how this differs from `ubuntu-latest`? It used to be a much clearer syntax

Reply:
Resolved as belongs to same topic of adding more comments to these files in general.